### ISSUE TITLE
Why it works failed on the GO2

### ISSUE THREAD

**Issue by Zedzou** [unitreerobotics/point_lio_unilidar#11]
I use the ros1_bridge to connect the ros2 of the GO2 and the ros1, but when I try run the point_lio_unilidar, the path of the point_lio_unilidar drifts badly, and when the go2 stop, the path still go on.

**Comment by lingbomeng** [comment]
The work environment on go2 is too severe for this lidar, especially the robot vibration and lidar mount pose. Most of the laser points raycast on the near front floor rather than the sorrunding environment, which leads to the failure of point-lio since it can not see enough information of it's periphery.

**Comment by ereshkigal224** [comment]
no，it is because the imu on the go2 is set upside down.So u need to turn it back.

**Comment by Robotics-SaintDiana** [comment]
@Zedzou Hello!
Sorry to bother you. I'm also trying to run point_lio_unilidar on a GO2, and I'm experiencing severe drift in the path and axes just as you mentioned. Could you please share how you managed to fix it? Thank you very much!

**Comment by lukeyang117** [comment]
> @ereshkigal224 Sorry to boter you and i just wanna know do you know how to turn it back?
 ##

**Comment by lukeyang117** [comment]
@ereshkigal224 sorry to bother you.Do you know how to turn it back?

**Comment by GeorgeKonno** [comment]
i agree to @ereshkigal224  seems the imu to lidar tran_r is not rpy[000]

**Comment by shlyang** [comment]
I've changed the `extrinsic_R` in `unilidar_l1.yaml` according to the radar link in [go2_description.urdf](https://github.com/Unitree-Go2-Robot/go2_description/blob/humble/urdf/go2_description.urdf) (rpy: [0, 2.8782, 0])
But it still doesn't work, when the go2 stop, the map moves like flying.

**Comment by jhaiq** [comment]
Has anyone known how to solve the same problem?

**Comment by babizhu** [comment]
same problem

**Comment by alitekes1** [comment]
@jhaiq  @babizhu i saw same problem and i cannot fixed it. but you can use octomap for mapping by use go2's own odometry data.
